.gitignore

```diff
@@ -7,3 +7,4 @@ doc
 pkg
 spec/reports
 tmp
+.idea
```

Comment:
I think it may make more sense to add this to your containing project’s `.gitignore` instead of here?